Guidelines.md

```diff
@@ -28,14 +28,8 @@ In general, dots are used to separate modifiers and subjects. For example, if yo
 represents, you might call that attribute `histogram.bucket`.
 * When generating groups of metrics that should be visualized together, the metric names for that group SHOULD be the customer-provided metric named, suffixed with name for the type of group, separated by a `.`.
   For example, if you are generating a set of gauges that represent a histogram, you would suffix the customer-provided metric name with "`.buckets`".
-
-### Common patterns:
 * All metric names SHOULD start with the customer-provided metric name that the metric library provides to the exporter.
```

Comment:
I might update this line while we're in here to say "customer or instrumentation-provided metric name"